src/records/Project.js

```diff
@@ -14,20 +14,23 @@ export default class Project extends Record({
   enabledLibraries: new Set(),
   hiddenUIComponents: new Set(),
   updatedAt: null,
+  readme: '',
```

Comment:
I don’t feel strongly but I think `instructions` might be a more straightforward description of the intent of this?